### ISSUE TITLE
[FR] Ability to delete cards

### ISSUE THREAD

**Issue by rebehe** [rebehe/kanban#2]
Give users the ability to delete cards from their list (possible use case: task is no longer needed).

**Comment by garbageman** [comment]
Did we make a decision on how we would want this to be implemented? I could add a button for a user to click in order to delete the card. We could also do a similar thing where if you hold on to the card for long enough a button to delete it will appear, like how icons work on iOS.

**Comment by garbageman** [comment]
I have card deletion functioning right now but it's just a button on each card, there should be a follow on to revisit how we want the user to interact with the card before deleting it. Functionally it is good to go.

**Comment by rebehe** [comment]
I tested this on master and tried creating a list of three cards and deleting the second card. Instead of the second card getting deleted, I saw the first card get deleted. I tried this again and tried deleting the third card this time, but also saw the first card get deleted. 

My hypothesis is that the current functionality will always delete the *first* card. I would guess that this has something to do with the way you're getting the index of the view.
